src/AcmExpandable/AcmExpandableCard/AcmExpandableCard.tsx

```diff
@@ -19,20 +19,32 @@ const useStyles = makeStyles({
     },
 })
 
+const onToggle = (acmCardID: string, open: boolean, setOpen: (open: boolean) => void) => {
+    setOpen(!open)
+    if (localStorage.getItem(acmCardID) === 'show') {
+        localStorage.setItem(acmCardID, 'hide')
+    } else {
+        localStorage.setItem(acmCardID, 'show')
+    }
+}
+
 export function AcmExpandableCard(props: {
     title: string
     children: React.ReactNode
     className?: string
     id?: string
 }) {
-    const [open, setOpen] = useState<boolean>(true)
+    const acmCardID = window.location.href + props.id
+    localStorage.getItem(acmCardID) ? localStorage.getItem(acmCardID) : localStorage.setItem(acmCardID, 'show')
```

Comment:
Can it be `localStorage.getItem(acmCardID) ?? localStorage.setItem(acmCardID, 'show')`?

Reply:
Changed this, thanks!